sources/quest_2/chapter_2/step_9/step_9_target.move

```diff
@@ -0,0 +1,15 @@
+module robinson::my_shore {
+
+    struct GlobalData has key {
+        nb_tree: u8,
+        has_river: bool,
+        shore_location: address,
+        animal: vector<u8>,
+    }
+
+    struct House has store{
+        no_of_members : u8,
+        house_location : address,
+        house_owner : address,
+    }
```

Comment:
Little space after `store`, it's a small detail for the display form 

```
struct House has store {
    no_of_members : u8,
    house_location : address,
    house_owner : address,
}
```

Reply:
Noted!